### PR TITLE
Add HeaderGuest

### PR DESCRIPTION
**Pull request by Bicik89**
new branch because of new concept and rebased to master. 
Also added register line in /login because header in normal width only has Login option
will delete organism one branch

### CODE REVIEW

File: src/components/pages/Login.vue [hunk 1]
```diff
@@ -32,4 +39,9 @@ h1 {
   margin-top: 1%;
 }
 
+span {
```
Comment (242370) [inline src/components/pages/Login.vue:49]: Add COLORS.TEXT_PRIMARY and FONT_SIZES.IMPORTANT

File: src/components/pages/Login.vue [hunk 2]
```diff
@@ -3,6 +3,7 @@
     <Header />
     <h1>Zaloguj się</h1>
     <FormLogin />
+    <span @click="onRegister">Nie posiadasz konta? Zarejestruj się</span>
```
Comment (242370) [inline src/components/pages/Login.vue]: Add exclamation point at the end

File: src/components/pages/Menu.vue [hunk 3]
```diff
@@ -1,30 +1,32 @@
...
-        <span @click="onOffersClick">Oferty</span>
-        <hr/>
-        <span @click="onMapClick">Mapa</span>
-        <hr/>
-        <span @click="onAddOfferClick">Nowa oferta</span>
-        <hr/>
-        <span @click="onAboutUsClick">O nas</span>
-        <hr/>
+        <span v-if="isLoggedIn" @click="onLogout">Wyloguj się</span>
+        <span v-else @click="onRegister">Zarejestruj się</span>
```
Comment (242370) [inline src/components/pages/Menu.vue:28]: When user is not logged in he doesn't have access to the Menu component, so remove this tag and all the logic related

**Review by 242370 — CHANGES_REQUESTED** [review]

File: src/components/pages/Login.vue (hunk 1 above)
Comment (Bicik89) [reply src/components/pages/Login.vue:49]: Font is now calculated, when full width it has IMPORTANT size

**Review by 242370 — APPROVED** [review]

**Pull request merged**